### DISCUSSION TITLE
Populate selection options from table [Apr 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1642] [Discord Threads]
> 🙋‍♂️ **sakibulk**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### sakibulk@discord I am sorry. We don't support it at the moment  
#  
> 🙋‍♂️ **VA-Ohm**  
#### will it be added at some point? I am just needing that option too  
#  
> 💁 **o1lab**  
#### yes, please stay tuned for the upcoming release. This will be supported.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959640801690935317)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
